concord-ui/src/components/systemstatstable.js

```diff
@@ -0,0 +1,201 @@
+import React, { Component } from 'react';
+import { getSystemStats } from '../api';
+import {
+  DataTable,
+  DataTableBody,
+  DataTableCell,
+  DataTableContent,
+  DataTableHead,
+  DataTableHeadCell,
+  DataTableRow
+} from 'rmwc/DataTable';
+import '@rmwc/data-table/data-table.css';
+import { connect } from 'react-redux';
+
+export class SystemStatsTable extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      statsData: {},
+      labelData: [],
+      doneFirstSort: false,
+      reloadApiData: false
+    };
+  }
+
+  componentDidMount() {
+    this.props.dispatch(getSystemStats());
+  }
+
+  componentDidUpdate(oldProps) {
+    const newProps = this.props;
+    if (oldProps.loading !== newProps.loading && newProps.statsData) {
+      const data = newProps.statsData;
```

Comment:
Should be able to do some destructuring here. e.g.

const { totalPhrases, completedPhrases, phrasesWithConsensus } = newProps.statsData;

Then you won't have to keep referencing the parent node.